### ISSUE TITLE
Remove Release SVN code

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8323]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8323|
|Type|task|
|Created|2016-05-10 13:03:26Z|
|Branch|https://github.com/twisted/twisted/tree/nosvn-8323|

We only do releases on Git now, so we can get rid of the SVN-release parts.

Part of [#8322](https://github.com/twisted/twisted/issues/8322).

<details><summary>Searchable metadata</summary>

```
trac-id__8323 8323
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__nosvn_8323 nosvn-8323
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1462885406482716 1462885406482716
changetime__1463229154367038 1463229154367038
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Happy with this too. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

All good. Thanks!

happy to see this cleanup. great work :)

please merge.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 4f2533c1c2f7e9c6f5ef5edb9799b06b8f30dd54
```
#!CommitTicketReference repository="" revision="4f2533c1c2f7e9c6f5ef5edb9799b06b8f30dd54"
Merge nosvn-8323: Remove SVN support in release code

Author: hawkowl
Reviewer: adiroiban
Fixes: #8323
```
